### ISSUE TITLE
[KEYBOARD]: Ensure content structure may be navigated by tab order

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3552]
## Description

**[KEYBOARD]: Ensure content structure may be navigated by tab order**
The tab order skips relevant content included the current production design, and the expected behavior of the links with the carat don't open the options, instead going to a new page.

## Point of Contact

Jennifer

## Acceptance Criteria

As a keyboard user, I want to navigate content in the website by pressing Tab, Spacebar or Enter. These keypress actions should be consistent and reveal all the relevant information.

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Enter **`https://www.va.gov/pittsburgh-health-care/`** in browser
2. Navigate the page using on your keyboard Tab, Space, and Enter keys
3. Note what information is available through these options
4. Enter **`https://staging.va.gov/pittsburgh-health-care/`** in browser
5. Navigate the page using on your keyboard Tab, Space, and Enter keys
6. Note what information and context is available through these two different treatments, particularly on staging if you try to access the Locations dropdown. The pattern typically used would associate the word 'Locations' and the carat as one link, so the expectation is to use Space to open the options, instead of going to a different page.

**Comment by mmiddaugh** [comment]
Hold on this one until design review complete

**Comment by mmiddaugh** [comment]
This issue should be resolved as part of the design changes in [#7212](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7212). Will confirm once changes are available in staging.  Moving this to the Icebox until then.

**Comment by mmiddaugh** [comment]
@jenstrickland this one is ready for your review.

**Comment by jenstrickland** [comment]
This one is addressed with the redesign.
